src/librustc_mir/tcx/expr.rs

```diff
@@ -147,17 +159,21 @@ impl<'tcx> Mirror<'tcx> for &'tcx hir::Expr {
 
             hir::ExprUnary(op, ref arg) => {
                 if cx.tcx.is_method_call(self.id) {
-                    overloaded_operator(cx, self, ty::MethodCall::expr(self.id),
-                                        PassArgs::ByValue, arg.to_ref(), vec![])
+                    overloaded_operator(cx,
+                                        self,
+                                        ty::MethodCall::expr(self.id),
+                                        PassArgs::ByValue,
+                                        arg.to_ref(),
+                                        vec![])
                 } else {
                     // FIXME overflow
                     let op = match op {
                         hir::UnOp::UnNot => UnOp::Not,
                         hir::UnOp::UnNeg => UnOp::Neg,
                         hir::UnOp::UnDeref => {
-                            cx.tcx.sess.span_bug(
-                                self.span,
-                                "UnDeref should have been handled elsewhere");
+                            cx.tcx
+                              .sess
+                              .span_bug(self.span, "UnDeref should have been handled elsewhere");
```

Comment:
@nrc @marcusklaas perhaps `(` are another case where we ought to consider a line-break? the original looks much better to me here.